src/get-started/codelab.md

```diff
@@ -99,6 +100,14 @@ other browsers. For more information,
 see [Building a web application with Flutter][]
 and [Write your first Flutter app on the web][codelab-web].
 
+Also, Flutter apps can compile for desktop.
+When you are developing Flutter apps in Windows,
```

Comment:
Consider re-working this to be about all three desktop OS platforms.

Reply:
I reworded to make it more generic